src/inc/WindowingBehavior.h

```diff
@@ -0,0 +1,10 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+--*/
+
+#pragma once
+constexpr int64_t WindowingBehaviorUseCurrent{ 0 };

```

Comment:
I don't honestly know how these symbols work when included in multiple translation units in the same library. Carlos' question about static does stick in my mind.

Reply:
https://stackoverflow.com/a/50489130

I don't know if that answers the question but it sounds like I'm doing it right?